modules/auxiliary/scanner/http/git_scanner.rb

```diff
@@ -0,0 +1,126 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::WmapScanServer
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name'        => 'HTTP Git Scanner',
+      'Description' => 'Detect git directories and files and analize its content.',
+      'Author'       => ['t0nyhj'],
+      'License'     => MSF_LICENSE
+    )
+
+    register_options(
+      [
+        OptString.new('PATH', [ true,  "The test path to .git directory", '/'])
```

Comment:
Use TARGET_URI as explained here: https://github.com/rapid7/metasploit-framework/wiki/How-to-Send-an-HTTP-Request-Using-HTTPClient